csunplugged/topics/models.py

```diff
@@ -71,12 +54,12 @@ class Meta:
         ordering = ["number", "name"]
 
 
-class LearningOutcome(models.Model):
+class LearningOutcome(TranslatableModel):
     """Model for learning outcome in database."""
 
     #  Auto-incrementing 'id' field is automatically set by Django
     slug = models.SlugField(max_length=80, unique=True)
-    text = models.CharField(max_length=200, unique=True)
+    text = models.CharField(max_length=200, default='')
```

Comment:
Is `default=''` required? Is this the same as `blank=True` (which I thought was default for string fields).

Reply:
From our discussion on Slack, this is required for the translation package.

Reply:
TL;DR Yes, `default=''` is required.

I thought exactly the same when I first implemented it, and was baffled by the fact that, without `default=''`, I was getting back `None` on unpopulated fields rather than `''`. This was at odds with the documentation, which says that generally no value for a  `TextField`/`CharField` is represented by an empty string on the database level, instead of null.

Turns out, `django-modeltranslation` sets `null=True` on the extra fields it creates for each language (i.e. `content_en`, `content_de` will have null=True even if the original `content` did not). This means `default=''` is required to force those translation fields to store no value as the empty string rather than null.

Reply:
Relevant issue on django-modeltranslation discussing this behavior - deschler/django-modeltranslation#144